src/behavioural-events/events.ts

```diff
@@ -0,0 +1,46 @@
+import { v4 as uuidv4 } from "uuid";
+import { type Trace } from "./trace";
+
+type EventType = "rcb_sdk_initialized";
+
+export abstract class BaseEvent {
+  public readonly id: string;
+  public readonly timestamp: number;
```

Comment:
Note: Using `timestamp` with ms from epoch time for consistency with customer center and paywall events.